ecs-agent/tcs/handler/handler_test.go

```diff
@@ -345,7 +346,7 @@ func TestConnectionInactiveTimeout(t *testing.T) {
 		testCreds,
 		testCfg,
 		deregisterInstanceEventStream,
-		50*time.Millisecond,
+		5*time.Second,
```

Comment:
Q: I think the reason we have this to a smaller value is to get the result faster. 5s is not long for me either but want to check is there any reason we have to change this?